src/recordlinker/linking/mpi_service.py

```diff
@@ -107,3 +107,25 @@ def insert_blocking_keys(
     if commit:
         session.commit()
     return values
+
+def get_all_algorithm_labels(session: orm.Session) -> list[str]:
+    """
+    Gets a list of algorithms from the MPI database.
+    returns: list of all labels column from algorithms table
+    """
+    algorithms_list = session.query(models.Algorithm.label).all()
+
+    return [algorithm[0] for algorithm in algorithms_list]
+
+def get_algorithm_by_label(session: orm.Session, label: str) -> models.Algorithm | None:
+    """
+    Gets a single algorithm by searching for the unique label
+    returns: algorithm object
+    """
+    if not label:
+        algorithm = session.query(models.Algorithm).filter(models.Algorithm.is_default == True).first() # noqa: E712
```

Comment:
I think this should be a `get()`, instead of a `filter()`.  There shouldn't be a situation where there is not one, and only one row, with is_default == True.  If it's missing or if there are more than one row with is_default set to true, I think we should be raising an error.

Reply:
I went ahead and switched these to `select` as well with `session.scalar`. You can do a `session.get(...)` but I think you can only use the primary key in that case.

Reply:
You're right, [get()](https://docs.sqlalchemy.org/en/20/orm/session_api.html#sqlalchemy.orm.Session.get) only works with primary keys, we'll have to build in this logic ourselves or put a helper function on the Algorithm class.